src/client/styles/Grid.ts

```diff
@@ -229,7 +229,7 @@ export const gridItem = (spanDefinition?: SpanDefinition) => {
 export const gridItemColumnConsents: SpanDefinition = {
   TABLET: { start: 2, span: 10 },
   DESKTOP: { start: 2, span: 10 },
-  LEFT_COL: { start: 2, span: 10 },
+  LEFT_COL: { start: 2, span: 12 },
```

Comment:
with the changes to the column width, this need changing so that they spread across the correct number of columns